src/partial-result-stream.js

```diff
@@ -104,6 +108,10 @@ function partialResultStream(requestFn) {
 
       rowChunks = [];
 
+      if (options.toJSON) {
+        formattedRows = formattedRows.map(exec('toJSON'));
```

Comment:
Although having `toJSON` available on `row` instances is great, having a static `toJSON` method would be cool too, since it's not dependent on the row's content. It would save you the trouble of using the `methmeth` library 😉 `formattedRows.map(Spanner.toJSON)`

Reply:
I tried going that route, but now that `options.toJSONOptions` is supported, it forced an inline function to wrap the `toJSON()` call anyway. So, now we have an easier to look at:

```js
formattedRows.map(exec('toJSON', options.toJSONOptions));
```